src/utils/talks/converter.ts

```diff
@@ -0,0 +1,62 @@
+//#region > Imports
+//> Models
+// Contains the talk model
+import { Talk } from "../../reducer/database/models";
+//#endregion
+
+//#region > Functions
+/**
+ * @function
+ * @param rawData
+ * @description Converter for talk data from the Github APIv3
+ */
+async function run(rawData: any) {
+  console.log(rawData);
```

Comment:
Is this console.log required?

Reply:
Right. I will remove it together with a bugfix.